controllers/helpers.go

```diff
@@ -101,3 +101,13 @@ func readRequest(w http.ResponseWriter, r *http.Request, errorNotifier errornoti
 
 	return requestBody, contentStoreRequest
 }
+
+func stripAccessLimitingMetadata(body []byte) []byte {
+	var unmarshalled map[string]interface{}
+	json.Unmarshal(body, &unmarshalled)
```

Comment:
We're ignoring any errors here.  I realise that if the request body is invalid JSON, this will already be caught by `readRequest`, so it should never error here.  I think it's probably worth a comment explaining that.